architecture/core_concepts/routes.adoc

```diff
@@ -817,11 +819,32 @@ resolution order (oldest route wins).
 ====
 
 
-[[route-specific-timeouts]]
-== Route-specific Timeouts
+[[route-specific-annotations]]
+== Route-specific Annotations
 
-By default, a server-side timeout is set across all routes. It is possible to
-change that timeout using an annotation in the route configuration.
+Using environment variables as defined in xref:env-variables[Configuration Parameters],
+a router can set the default options for all the routes it exposes. An individual route can
+override some of these defaults by providing specific configuration in its annotations.
+
+
+*Route Annotations*
+
+For all the items outlined in this section, you can set annotations on the
+*route definition* for the route to alter its configuration
+
+.Route Annotations
+[cols="2*", options="header"]
+|===
+|Variable | Description
+|`*haproxy.router.openshift.io/balance*`| sets the load balance algorithm. Available options are `source`, `roundrobin`, or `leastconn`.
```

Comment:
The first letter of the description should be uppercase to match the other table.

Should we indicate the ENV that provides the default (perhaps in a new column)

load-balancing (or make the pervious one match this case)